src/bg/help/faq/README.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Често задавани въпроси
+lang: bg-BG
+---
+
+# Често задавани въпроси
+Ако имате въпроси относно приложението и плановете ни за него, или се нуждаете от помощ с инсталацията, вижте упътването и често задаваните въпроси. Ако още не намирате отговор на проблема си, може да се включите в Discord групата и да попитате.
+
+## Основна информация
+
+### Защо направихте това приложение?
+Липсата на качествени приложения за четене на манга на **iOS** без _натрапчиви_ реклами или поддръжка на **MangaDex**.
+
+### Какъв бизнес модел ще предприемете?
+Няма настоящи планове за формален бизнес модел. Докато Patreon покрива цената на единичен Apple Developer акаунт, печалбата е вторична. Бихме се радвали на помощ с разработка на приложението, но няма да има такси за ползването на Paperback.
+
+### Кога възнамерявате да пуснете приложението?
+Има публичен билд на приложението в [GitHub](https://github.com/Paperback-iOS/app/releases). Срещаме проблеми с придобиването на Apple Developer акаунт, но се надяваме да пуснем тестови билдове скоро. След тях очакваме да пуснем Paperback в App Store. За упътвания за инсталацията вижте:
+
+ * [Инсталиране на Paperback](/help/guides/getting-started)
```

Comment:
```suggestion
 * [Инсталиране на Paperback](/bg/help/guides/getting-started)
```

Reply:
Thank you for the update, some things slip.